web/src/components/AssertionForm/DurationFields.tsx

```diff
@@ -0,0 +1,61 @@
+import {FormInstance, Input, Select} from 'antd';
+import {TAssertion} from '../../types/Assertion.types';
+import {IValues} from './AssertionForm';
+import * as S from './AssertionForm.styled';
+
+type Duration = 'ms' | 's';
```

Comment:
It wont only support `ms` and `s`. It should support `ns|μs|ms|s|m|h`

Reply:
made this a enum instead